_locales/en/messages.json

```diff
@@ -242,14 +242,21 @@
   "extensionHasUpdated": {
     "message": "Scratch Addons has updated to v$1"
   },
-  "extensionUpdateInfo1_v1_32": {
-    "message": "New addon: \"move sprite to front layer\", which lets you Shift+Click a sprite within the sprite pane to move it to the front (top layer) of the stage. Go to $1 to enable it."
+  "extensionUpdateInfo1_v1_33": {
+    "message": "New addon: \"highlight project creator in comments\", which adds a badge next to the project creator in project comments sections. Go to $1 to enable it."
   },
   "scratchAddonsSettings": {
     "message": "Scratch Addons settings"
   },
-  "extensionUpdateInfo2_v1_32": {
-    "message": "Other new addons added this update are \"faster project loading\", \"follow topics after posting\", \"necropost highlighter\", and \"remember collapsed forum categories\"."
+  "extensionUpdateInfo2_v1_33": {
+    "message": "Multiple addons have been adjusted to work with the upcoming Scratch changes. This update also brings performance improvements, increased storage for addon settings, and more."
+  },
+  "readTheBlogPost": {
+    "message": "Read the $1 for more information.",
+    "description": "The placeholder is \"blog post\""
+  },
+  "blogPost": {
+    "message": "blog post"
```

Comment:
These 2 strings above are unused in the code as of now, but it's a good idea to send them for translation.

Reply:
I think just adding it with the other links is fine, but in the notification text it will get more attention.